packages/url_launcher/url_launcher/pubspec.yaml

```
@@ -32,11 +29,8 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.3.0
-  # Temporary path dependencies to allow moving Android and iOS implementations.
-  url_launcher_android:
-    path: ../url_launcher_android
-  url_launcher_ios:
-    path: ../url_launcher_ios
+  url_launcher_android: ^2.0.13
+  url_launcher_ios: ^2.0.13
```

One more little mistake ;) The major number also needs to change. The version on pub.dev is `6.0.13`:

```suggestion
  url_launcher_android: ^6.0.13
  url_launcher_ios: ^6.0.13
```

---

Yep, I was focused on the patch version being wrong and totally missed that the major version was completely different from the other implementation packages.